src/containers/SubjectViewer.jsx

```diff
@@ -4,13 +4,21 @@ import { connect } from 'react-redux';
 import { getSubjectLocation } from '../lib/get-subject-location';
 import { subjectError, SUBJECT_STATUS } from '../ducks/subject';
 import SubjectError from '../components/SubjectError';
+import { Utility } from '../lib/Utility';
 
 import {
-  resetView, updateImageSize, updateViewerSize
+  resetView, setTranslation,
+  updateImageSize,
+  updateViewerSize, SUBJECTVIEWER_STATE
 } from '../ducks/subject-viewer';
 
 import SVGImage from '../components/SVGImage';
 
+const INPUT_STATE = {
```

Comment:
This should eventually moved to a general constants lib, perhaps along with various other states. (e.g. FETCH_STATE to replace PROJECT_STATE, SUBJECT_STATE, etc since they're all the same IDLE/FETCHING/SUCCESS/FAILURE)

(But not replacing SUBJECTVIEWER_STATE or similar, which is unique to the specific duck.)